features/lorawan/lorastack/mac/LoRaMac.cpp

```diff
@@ -83,7 +83,7 @@ LoRaMac::LoRaMac()
       _prev_qos_level(LORAWAN_DEFAULT_QOS),
       _demod_ongoing(false)
 {
-    memset(&_params, 0, sizeof(_params));
+    _params = loramac_protocol_params();
```

Comment:
You can go one step better by simply putting `_params()` in the member-initializer-list above.

And a lot of the `_params.keys.xxxx = NULL` or other settings to 0/false below can also go.

Either or both of those may lead to a code size saving.

Reply:
There was actually no code size saving but it is neat.
https://github.com/ARMmbed/mbed-os/compare/d914234d18cb510914926687a7a39bcfc085e287..5d2f4528856ad5730ab1012984266a3ae52d6185